commit 9d0adf249827dde9e1c0d1c19513cf6018ceb34e
Author: Cole Robinson <crobinso@redhat.com>
Date:   Wed May 19 15:57:56 2010 -0400

    daemon: A few initscript corrections
    
    Fedora bug https://bugzilla.redhat.com/show_bug.cgi?id=565238
    
    - Avahi service is called 'avahi-daemon'
    - chkconfig descriptions must use \ for line continuations

diff --git a/daemon/libvirtd.init.in b/daemon/libvirtd.init.in
index d4dc98b..809433e 100644
--- a/daemon/libvirtd.init.in
+++ b/daemon/libvirtd.init.in
@@ -9,7 +9,7 @@
 # Should-Start: $named
 # Should-Start: xend
 # Should-Start: hal
-# Should-Start: avahi
+# Should-Start: avahi-daemon
 # Required-Stop: $network messagebus
 # Should-Stop: $named
 # Default-Start: 3 4 5
@@ -24,8 +24,8 @@
 # libvirtd:   guest and virtual network management daemon
 #
 # chkconfig: 345 97 03
-# description:  This is a daemon for managing guest instances
-#               and libvirt virtual networks
+# description:  This is a daemon for managing guest instances \
+#               and libvirt virtual networks \
 #               See http://libvirt.org
 #
 # processname: libvirtd
